app/pods/publications/publication/controller.js

```diff
@@ -102,28 +119,32 @@ export default class PublicationController extends Controller {
 
   @action
   setPublicationBeforeDate(event) {
-    this.publicationBeforeDate = new Date(event.target.value);
+    this.publicationBeforeDate = new Date(event);
     this.model.set('publishBefore', this.publicationBeforeDate);
     this.model.save();
   }
 
   @action
   setTranslationDate(event) {
-    this.translationDate = new Date(event.target.value);
+    this.translationDate = new Date(event);
     this.model.set('translateBefore', this.translationDate);
     this.model.save();
   }
 
+  // TODO change this
   @action
   setPublicationStatus(event) {
-    if (event.target.value === 'te publiceren') {
+    if (event.target.value === 'Te publiceren') {
       this.showToPublish = true;
       this.published = false;
     } else {
       this.showToPublish = false;
       this.published = true;
     }
     this.publicationStatus = event.target.value;
+    // CONFIG gebruiken en findRecord('publication-flow, config. ... . id)
+    // Wat er hier gebeurt is een string saven in een relatie
+    // kijk desnoods naar 'formally-ok' manier van werken
     this.model.set('status', this.publicationStatus);
     this.model.save();
```

Comment:
Wat je hier doet lijkt me gedeeltelijk fout. Je zou geen tekst moeten laten doorgeven maar de ID van de values die in de config staan. Daarna moet je de config ophalen uit de store en die wegschrijven in de publicatiestatus relatie.

Wat jij nu doet lijkt me een effectieve string value wegschrijven naar het model, terwijl status een object/relatie is.

Reply:
De opties in de dropdown zijn nu tekst velden, deze opties kunnen we opbouwen als objecten met de waarden van de config. Dan moet je in de save functie ook niet controleren op de value. Dan is de inkomende value vb het id of de URI van de status ipv tekst.